client/app/utils/message_utils.coffee

```diff
@@ -175,11 +185,12 @@ module.exports = MessageUtils =
             when null
                 @setMessageAsDefault options
 
-        # remove my address from dests
-        notMe = (dest) -> return dest.address isnt myAddress
-        message.to = message.to.filter notMe
-        message.cc = message.cc.filter notMe
-        return message
+        # # remove my address from dests
```

Comment:
Dont leave commented code, if you really really think we will need it in the future, leave a comment with a git commit number.